applications/luci-app-sshtunnel/htdocs/luci-static/resources/view/sshtunnel/ssh_servers.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+'require form';
+'require fs';
+'require uci';
+'require ui';
+'require view';
+
+var allSshKeys = {};
+
+return view.extend({
+	load: function () {
+		return L.resolveDefault(fs.list('/root/.ssh/'), []).then(function (entries) {
+			var tasks = [];
+			for (var i = 0; i < entries.length; i++) {
+				if (entries[i].type === 'file' && entries[i].name.match(/\.pub$/)) {
+					tasks.push(Promise.resolve(entries[i].name));
+				}
+			}
+			return Promise.all(tasks);
+		});
+	},
+
+	render: function (data) {
+		var sshKeys = _splitSshKeys(data);
+		if (sshKeys.length === 0) {
+			ui.addNotification(null, E('p', _('No SSH keys found, <a href="./ssh_keys">generate a new one</a>')), 'warning');
+		}
+
+		var m, s, o;
+
+		m = new form.Map('sshtunnel', _('SSH Tunnels'),
+			_('This configures <a href="https://openwrt.org/docs/guide-user/services/ssh/sshtunnel">SSH Tunnels</a>.')
```

Comment:
Please separate this link out to a `.format()`
This is how I did it for another comment:
`			_('Discard upstream responses containing <a href="%s">RFC1918</a> addresses.').format('https://datatracker.ietf.org/doc/html/rfc1918'));
`